src/components/WebChat/WebChat.css

```diff
@@ -0,0 +1,130 @@
+.webchat-container{
+  display: flex; 
+  flex-direction: column;
+  position: fixed;
+  right: 0;
+  bottom: 0;
+  bottom: 20px;
+  box-shadow: 5px 5px 20px #fff;
+}
+
+.webchat-panel-inner{
+  flex-direction: column;
+}
+
+.webchat-bar, .webchat-open__btn{
+  height: 45px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  background-color:#6e267b;
+  padding: 40px;
+}
+
+.webchat-bar-spacing, .webchat-btn-spacing{
+  padding: 40px;
+}
+
+.webchat-window{
+  flex-direction: column;
+  background-color: WhiteSmoke;
+  height: 300px;
+  overflow-y: auto;
+}
+
+.webchat-window, .webchat-bar, .webchat-open__btn, .webchat-message-submit{
+  width: 400px;
+  padding: 20px;
+  margin-left: auto;
+  color: white;
+  flex-basis: auto;
+}
+
+.webchat-message-submit{
+  justify-content: center;
+  padding-bottom: 20px;
+  padding-top: 20px;
+  background-color: #F8F8F8;
+  flex-basis: auto;
+  border-bottom-right-radius: 5px;
+  border-bottom-left-radius: 5px;
+}
+
+.webchat-toggle--hide{
+  display: none;
+}
+
+.webchat-toggle--show{
+  display: flex;
+}
+
+.webchat-chatbubble-usera, .webchat-chatbubble-userb{
+  border: none; 
+  border-radius: 8px;
+  padding: 15px;
+  width: 170px;
+  min-height: 10px;
+  max-height: fit-content;
+  margin: 15px 7px 15px 7px;
+  font-size: 11px;
+  text-align: left;
+}
+
+.webchat-chatbubble-usera{
+  background: #E8E8E8; 
+  color: black;
+}
+
+.webchat-chatbubble-userb{
+  background: blueviolet;
+  color: white;
+  align-items: center;
+  display: flex;
+}
+
+.webchat-message-usera{
+  align-self: flex-start; 
+  flex-direction: row;
+}
+
+.webchat-message-userb{
+  align-self: flex-end;
+  flex-direction: row-reverse;
+}
+
+.webchat-message-usera, .webchat-message-userb{
+  display: flex;
+}
+
+.message-input, .message-submit__btn{
+  border: 0px none;
+  height: 40px;
+  margin: 7px;
+}
+
+.message-input{
+  width: 270px;
+}
+
+.message-submit__btn{
+  width: 40px;
+}
+
+.webchat-dot{
+  height: 20px;
+  width: 20px;
+  background-color: #808080;
+  border-radius: 50%;
+  display: flex;
+  margin-top: 17px;
+}
+
+input{
```

Comment:
the entire input in your application cud potentially be overwritten by this line.